sdk/search/Microsoft.Azure.Search/changelog.md

```diff
@@ -0,0 +1,227 @@
+# GA SDK Releases
+
+Every minor version release targets the same (GA) API version as that of the corresponding major version release. API version targeted by the major version releases are listed when appropriate.

```

Comment:
This would read a bit better if you mention that it's the REST API version. Also, the plurality of API version in the second sentence is unclear. It should be one of:

> REST API versions targeted by...

or

> The REST API version targeted by...